xinference/device_utils.py

```diff
@@ -17,14 +17,25 @@
 
 import torch
 
-DeviceType = Literal["cuda", "mps", "xpu", "npu", "mlu", "cpu"]
+DeviceType = Literal["cuda", "mps", "xpu", "vacc", "npu", "mlu", "cpu"]
 DEVICE_TO_ENV_NAME = {
     "cuda": "CUDA_VISIBLE_DEVICES",
     "npu": "ASCEND_RT_VISIBLE_DEVICES",
     "mlu": "MLU_VISIBLE_DEVICES",
+    "vacc": "VACC_VISIBLE_DEVICES",
 }
 
 
+def is_vacc_available() -> bool:
```

Comment:
Can we cache the result of this function with `@cache`? I think this function may be called may times to cause waste compute.